packages/headless/src/features/question-answering/question-answering-analytics-actions.ts

```diff
@@ -67,15 +67,13 @@ export const logLikeSmartSnippet = (): CustomAction =>
       return client.makeLikeSmartSnippet();
     },
     analyticsType: 'Qna.SubmitFeedback',
```

Comment:
These analyticsType should become 'Qna.AnswerAction'.
Snapshots to be updated accordingly.